src/puppetlabs/trapperkeeper/services/status/status_service.clj

```diff
@@ -33,6 +33,11 @@
       (add-ring-handler this handler))
     context)
 
+  (stop [this context]
+    (status-core/dissoc-status-context! (:status-fns context)
+                                        status-core/status-service-name)
+    context)
```

Comment:
the tests are structured that way for a reason; it's doing testing between the file sync client and storage services and needs to make the storage service unreachable momentarily. It's possible to structure that test in another way but it's more than just switching to `restart`.